.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,39 @@
+name: "Unit Tests"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
+
+      - name: Install Library
+        run: pip install .
+
+      - name: Install Dependencies
+        run: pip install -r test-requirements.txt
+
+      - name: Run Unit Tests
+        run: pytest -vvv tests/unit
+
+      - name: Assert no file changes
```

Comment:
I would assume this line is only relevant for snapshot tests?

Reply:
in the past, running GX has sometimes unexpectedly caused directories to be created (my machine is littered with `expectations` folders..), so i figured this might actually have some value

Reply:
Ah, I see. Fair enough!

Reply:
😂 you know, i think you're right, this doesn't really add a lot of value. removed in [remove diff](https://github.com/great-expectations/gx-airflow-provider/pull/2/commits/20c55f76b2a4f7e799a3aff96e37ebde36b76b0e)